_data-prepper/pipelines/configuration/processors/parse-xml.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: parse_xml
+parent: Processors
+grand_parent: Pipelines
+nav_order: 83
+---
+
+# parse_xml
+
+The `parse_xml` processor parses XML data for an event.
+
+## Configuration
+
+You can configure the `parse_xml` processor with the following options.
+
+| Option | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `source` | No | String | The field in the `event` that is parsed. Default value is `message`. |
+| `destination` | No | String | The destination field of the parsed XML. Defaults to the root of the `event`. Cannot be `""`, `/`, or any white-space-only `string` because these are not valid `event` fields. |
```

Comment:
```suggestion
| `destination` | No | String | The destination field of the parsed XML. Defaults to the root of the `event`. Cannot be `""`, `/`, or any white-space-only string because these are not valid `event` fields. |
```